test/unit/modules/helpers/get-n-parents-spec.js

```diff
@@ -117,17 +135,15 @@ describe('RecordingLibrary.helpers.getNParents', () => {
     ]);
   });
 
-  test('should return an empty array if there are no parents', () => {
-    expect(getNParents(p6)).toEqual([]);
-  });
-
```

Comment:
This is not possible in an actual DOM: valid `HTMLElement` nodes always have a parent, unless they are not attached to the document or are the `document` node themselves, and those aren't valid use-cases for our tracking.